examples/pinproctest/pinproctest.cpp

```diff
@@ -27,146 +27,122 @@
  *  libpinproc
  */
 #include "pinproctest.h"
+uint32_t board_id = 0;
 
 PRMachineType machineType = kPRMachineInvalid;
 
 /** Demonstration of the custom logging callback. */
 void TestLogger(PRLogLevel level, const char *text)
 {
-    fprintf(stderr, "TEST: %s", text);
-}
-
-PRResult LoadConfiguration(YAML::Node& yamlDoc, const char *yamlFilePath)
-{
-    try
-    {
-        std::ifstream fin(yamlFilePath);
-        if (fin.is_open() == false)
-        {
-            fprintf(stderr, "YAML file not found: %s\n", yamlFilePath);
-            return kPRFailure;
-        }
-        
-        yamlDoc = YAML::Load(fin);
-    }
-//    catch (YAML::ParserException& ex)
-//    {
-//        fprintf(stderr, "YAML parse error at line=%d col=%d: %s\n", ex.line, ex.column, ex.msg.c_str());
- //       return kPRFailure;
-//    }
-//    catch (YAML::RepresentationException& ex)
-//    {
-//        fprintf(stderr, "YAML representation error at line=%d col=%d: %s\n", ex.line, ex.column, ex.msg.c_str());
-//        return kPRFailure;
-//    }
-    catch (...)
-    {
-        fprintf(stderr, "Unexpected exception while parsing YAML config.\n");
-        return kPRFailure;
-    }
-    return kPRSuccess;
+    printf("TEST: %s", text);
 }
 
 void ConfigureAccelerometerMotion(PRHandle proc)
 {
     uint32_t readData[5];
 
-    PRReadData(proc, 6, 0x10D, 1, readData); 
-    printf("\nAccel chip id: %x\n", readData[0]);
-    fflush(stdout);
+    // Only the P3-ROC has an accelerometer.
+    if (board_id != P3_ROC_CHIP_ID) {
+        return;
+    }
```

Comment:
Accelerometer code was crashing with my P-ROC, so I added checks to ensure we only attempt accelerometer reads on P3-ROC.